tslint.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": [
+    "tslint-config-exmg"
+  ],
+  "rules": {
+    "ter-prefer-arrow-callback": [false]
```

Comment:
this rule is only for tests shouldn't be here

Reply:
done